tests/unit/ExtraproRegistration.spec.ts

```diff
@@ -34,30 +35,16 @@ describe('Extrapro Registration component', () => {
       homeIdentifier: '',
       homeIncorporationDate: '',
       homeLegalName: '',
-      mode: 'EXPRO'
-    }
-    const getters = {
-      isContinuationInAffidavitRequired: () => true,
-      getExistingBusinessInfo: () => businessMock
-    }
-
-    const store = new Vuex.Store({
-      getters
-    })
+      mode: 'EXPRO',
+      status: 'ACTIVE'
+    } as ExistingBusinessInfoIF
+    store.stateModel.entityType = CorpTypeCd.ULC_CONTINUE_IN
 
-    const wrapper = shallowMount(UploadAffidavit, {
-      store,
-      propsData: {
-        business: businessMock
-      }
-    })
+    const wrapper = wrapperFactory(ExtraproRegistration)
     await Vue.nextTick()
-    expect(wrapper.findComponent(UploadAffidavit).exists()).toBe(true)
 
-    // Check if the text content is rendered correctly
-    expect(wrapper.text()).toContain('Upload the affidavit from the directors.')
-    expect(wrapper.text()).toContain('Use a white background and a legible font with contrasting font colour')
-    expect(wrapper.text()).toContain('PDF file type (maximum 30 MB file size)')
```

Comment:
No need to test the sub-component here -- that's already done in a separate test suite.

In this test suite, we only test this component (and verify sub-components are rendered, if needed).